lib/swagger-mock-validator/spec-parser/parsed-spec.d.ts

```diff
@@ -60,8 +60,8 @@ interface ParsedSpecJsonSchemaBooleanKeywords {
 interface ParsedSpecJsonSchemaValue {
     additionalProperties?: boolean | ParsedSpecJsonSchema;
     enum?: any[];
-    exclusiveMaximum?: boolean;
-    exclusiveMinimum?: boolean;
+    exclusiveMaximum?: number;
+    exclusiveMinimum?: number;
```

Comment:
OpenAPI defines these as booleans, as did AJV@6.

But since https://json-schema.org/draft-06/json-schema-release-notes.html, it has been converted to a number. AJV followed suite.